src/validator/index.ts

```diff
@@ -0,0 +1,14 @@
+import { UserOperation } from "../type";
+
+export type ValidInputTypes = Uint8Array | bigint | string | number | boolean;
+
+export const isHexStrict = (hex: ValidInputTypes): boolean =>
+  typeof hex === "string" && /^((-)?0x[0-9a-f]+|(0x))$/i.test(hex);
+
```

Comment:
You may use `isHexStrict` from `web3-utils`, if you like:

```suggestion
import { isHexStrict } from "web3-utils";

import { UserOperation } from "../type";

export type ValidInputTypes = Uint8Array | bigint | string | number | boolean;

```

Reply:
Thx, I noticed that the `isHexStrict` function from `web3-utils` was marked as deprecated in the documentation. So, I've switched to using the equivalent function from `web3-validator`instead.